src/vs/workbench/contrib/preferences/browser/settingsTreeModels.ts

```diff
@@ -288,6 +289,14 @@ export class SettingsTreeSettingElement extends SettingsTreeElement {
 
 		return Array.from(extensionFilters).some(extensionId => extensionId.toLowerCase() === this.setting.extensionInfo!.id.toLowerCase());
 	}
+
+	matchesAnyFeature(featureFilters?: Set<string>): boolean {
+		const features = ['explorer', 'search', 'debug', 'scm', 'extensions', 'terminal', 'task', 'problems', 'output', 'comments', 'remote', 'timeline'];
```

Comment:
This shouldn't be hardcoded here, we can figure out how to share code with settingsLayout.ts

Reply:
not sure how to do with without adding a bunch of code.

Reply:
done.